src/redux/features/plantings/plantingsSelectors.ts

```diff
@@ -22,13 +23,13 @@ export const getTotalPlantsBySubzone = (plantings: PlantingSearchData[]) => {
   }, {});
 };
 
-export const getTotalPlantsBySite = (plantings: PlantingSearchData[]) => {
-  return plantings?.reduce((totalPlantsBySite: { [key: string]: number }, planting) => {
-    const plantingSiteId = planting.plantingSite.id;
+export const getTotalPlantsBySite = (plantingsSites: PlantingSiteSearchResult[]) => {
```

Comment:
@constanzauanini I made some changes to the selectors, there will be conflicts. You can override the conflicts related to this particular function.